lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
> Function -> format string -> low level function (.ess.eval, .ess.source etc).

No, the function returns a string ready for evaluation. The returned string does not need to be formatted.

> The long name ess-format-eval-command is not very appealing to me because 'format' reads as verb in there.

That's intended, it does format a function to send to the inferior based on a bunch of parameters.

> There might be that the current format commands are already modified by some users.

That could be a concern. Does anyone do that though? They can still write their own function.

> We use terms source, load and eval somewhat interchangeably.

I think the nomenclature is that load and eval are both about a form of sourcing. Load is about a file, eval about a command.

Where things get blurry is that we load files through eval for tramp performance... I was confused for a long time by this.